src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7300,6 +7300,12 @@ internal static string CombineVariableWithPartialPath(VariableExpressionAst vari
             var varPath = variableAst.VariablePath;
             if (varPath.IsVariable || varPath.DriveName.Equals("env", StringComparison.OrdinalIgnoreCase))
             {
+                if (varPath.UnqualifiedPath.Equals(SpecialVariables.PSScriptRoot, StringComparison.OrdinalIgnoreCase)
```

Comment:
Please confirm this will work well on case-sensetive file systems.

Reply:
> varPath.DriveName.Equals("env", StringComparison.OrdinalIgnoreCase)

Do we need to process PSScriptRoot for the condition too? If no I'd prefer to simplify nesting.